config.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/* eslint no-process-env: 0 */
+module.exports = {
+  env: process.env.NODE_ENV,
+  postcode: {
+    hostname: process.env.MOCK_POSTCODE === 'true' ?
+      `http://${process.env.LISTEN_HOST || '0.0.0.0'}:${process.env.PORT || 8080}/api/postcode-test` :
+      'https://postcodeinfo.service.justice.gov.uk'
+  }
+};
```

Comment:
education question for myself: What is going on with how the api is used with the live service and also the mock service? 

I see the hostname is set to use the live service or 8080/api/postcode-test based on the process arg but I don't understand how it works e.g.
1. where is this api/postcode-test, is it the mock-postcode.js?
2. api calls usually require api keys etc, where are these stores and used?